manifests/m/Microsoft/DevHome/0.701.323.0/Microsoft.DevHome.installer.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Microsoft.DevHome
+PackageVersion: 0.701.323.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.22000.0
+InstallerType: msix
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.3

```

```suggestion
  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.4
```
Based on the error, I'm guessing this might need to be updated.

---

Opened PR for `Microsoft.WindowsAppRuntime.1.4` so that the dependency is referenced properly once changes here get accepted:

- https://github.com/microsoft/winget-pkgs/pull/127763